src/fileUtils.ts

```diff
@@ -0,0 +1,15 @@
+import { Codacyrc } from "./model/CodacyInput"
+import { readFile } from "./promisified"
+
+export async function readJsonFile(file: string): Promise<string | undefined> {
+  try {
+    return await readFile(file, "utf8")
```

Comment:
`return await` seems redundant just to catch the error. Could this go directly on index.ts?

Reply:
`index.ts` is already full of stuff. I prefer to leave it here.